scss/app.scss

```diff
@@ -4,3 +4,49 @@
 @import "footer";
 
 @import "git_repo";
+
+h3.titlerepo {
+	color: #1976D2;
+}
+h5.subtitlerepo {
+	font-weight: normal;
+}
+h4.descriptionrepo {
+	font-size: 14px;
+	font-weight: normal;
+}
+h4.totalforkrepo {
+    font-weight: normal !important;
```

Comment:
if you want this to be `font-weight: normal`, why use `h4` element in the first place?